### PR TITLE
#160220034 Fix Travel Request Form

### PR DESCRIPTION
**Pull request by albert85**
#### What does this PR do?
Fix travel request form bugs

#### Description of Task to be completed?
- Change the "Cancel" label to "Clear"
- Indicate require field by adding a red asterisk symbol
- Disable the arrival date until departure date is selected
- Disable all dates prior to departure date

#### How should this be manually tested?
- Clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Run the server `make start`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#160220034](https://www.pivotaltracker.com/story/show/160220034)

#### Screenshots (if appropriate)
screen shot 2018-09-05 at 1 02 27 pm

#### Questions:
N/A

### CODE REVIEW

**Comment by samuelkubai** [comment]
@amarachukwu-agbo from your screenshot I see you haven't pulled changes from @amarachukwu-agbo's PR since that needs to be a dropdown and not a text field for the manager's input.

**Comment by samuelkubai** [comment]
@albert85 please update your PR title

File: src/components/Forms/FormsAPI/index.js [hunk 1]
```diff
@@ -18,14 +20,15 @@ class InputRenderer {
       inputProps.choices = this.formMetadata.buttonToggleOptions[name];
       return inputProps;
     case 'date':
+      // inputProps.minDate = minDate;
```
Comment (amarachukwu-agbo) [inline src/components/Forms/FormsAPI/index.js]: Please remove the comment

Comment (amarachukwu-agbo) [reply src/components/Forms/FormsAPI/index.js]: The implementation works as expected

File: src/components/Forms/NewRequestForm/FormFieldsets/TravelDetails.jsx [hunk 2]
```diff
@@ -10,20 +11,31 @@ class TravelDetailsFieldset extends Component {
...
 
+    let customPropsForDepartureDate,
+      customPropsForArrivalDate,
+      customPropsOtherDest;
+    customPropsOtherDest = { className: otherDestCustomClass };
+    customPropsForDepartureDate = { minDate: moment() };
+    customPropsForArrivalDate = {
+      disabled: !values.departureDate,
+      minDate: moment(values.departureDate),
+      placeholderText: 'select departure date first'
```
Comment (babadee001) [inline src/components/Forms/NewRequestForm/FormFieldsets/TravelDetails.jsx]: Can we have the placeholderText changed to **select arrival date** when departureDate has been filled/selected?

**Comment by babadee001** [comment]
LGTM @samuelkubai

**Pull request merged**